### PR TITLE
Fix Avalara caching.

### PR DESCRIPTION
**Pull request by fowczarek**
I want to merge this change because of fixing Avalara caching.

Currently, some queries take 15s because of caching Avalara requests not working. 

## If we need to calculate the price for one checkout line we should sed to Avalara the whole checkout!!!

# Impact

* [ ] New migrations
* [ ] New/Updated API fields or mutations
* [ ] Deprecated API fields or mutations
* [ ] Removed API types, fields, or mutations
* [ ] Documentation needs to be updated

# Pull Request Checklist

* [ ] Privileged queries and mutations are guarded by proper permission checks
* [ ] Database queries are optimized and the number of queries is constant
* [ ] Database migration files are up to date
* [ ] The changes are tested
* [ ] GraphQL schema and type definitions are up to date
* [ ] Changes are mentioned in the changelog

### CODE REVIEW

**Comment by patrys** [comment]
Task linked: [SALEOR-2681 Caching of Avalara request not working.](https://app.clickup.com/t/2549495/SALEOR-2681)

**Review by maarcingebala — APPROVED** [review]

**Review by IKarbowiak — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -52,6 +52,12 @@ All notable, unreleased changes to this project will be documented in this file.
     - `get_checkout_line_tax_rate`
     - `preprocess_order_creation`
   - additionally, `preprocess_order_creation` was extend with `lines_info` parameter
+- Fix Avalara caching - #7036 by @fowczarek; Introduced changes in plugin methods definitions:
```
Comment (korycins) [inline CHANGELOG.md:55]: Duplicated sentence

File: saleor/plugins/avatax/tests/test_avatax_caching.py [hunk 2]
```diff
@@ -0,0 +1,627 @@
...
+    # when
+    assert result == TaxedMoney(net=Money("72.2", "USD"), gross=Money("75", "USD"))
+
+    avalara_cache_key = CACHE_KEY + str(checkout.token)
+    mocked_cache.assert_called_with(avalara_cache_key)
+    mock_cache_set.assert_not_called()
+
+
+@override_settings(PLUGINS=["saleor.plugins.avatax.plugin.AvataxPlugin"])
+def test_calculate_checkout_total_save_avatax_respons_in_cache(
```
Comment (korycins) [inline saleor/plugins/avatax/tests/test_avatax_caching.py:59]: typo `respons`  ?

File: saleor/plugins/avatax/tests/test_avatax_caching.py [hunk 3]
```diff
@@ -0,0 +1,627 @@
...
+    # when
+    assert result == TaxedMoney(net=Money("64.07", "USD"), gross=Money("65", "USD"))
+
+    avalara_cache_key = CACHE_KEY + str(checkout.token)
+    mocked_cache.assert_called_with(avalara_cache_key)
+    mock_cache_set.assert_not_called()
+
+
+@override_settings(PLUGINS=["saleor.plugins.avatax.plugin.AvataxPlugin"])
+def test_calculate_checkout_subtotal__save_avatax_respons_in_cache(
```
Comment (korycins) [inline saleor/plugins/avatax/tests/test_avatax_caching.py:142]: Is `__` instead of `_` expected here?

**Review by korycins — APPROVED** [review]

**Pull request merged**